src/Application/Regulation/Command/DuplicateRegulationCommandHandler.php

```diff
@@ -35,6 +35,7 @@ private function duplicateRegulationOrderRecord(
         RegulationOrder $originalRegulationOrder,
     ): RegulationOrderRecord {
         $generalInfo = new SaveRegulationGeneralInfoCommand();
+        $generalInfo->category = $originalRegulationOrder->getCategory();
```

Comment:
Il faut aussi dupliquer le texte libre éventuel, non ?
```suggestion
        $generalInfo->category = $originalRegulationOrder->getCategory();
        $generalInfo->otherCategoryText = $originalRegulationOrder->getOtherCategoryText();
```